cli-feed-v3-2.json

```diff
@@ -25,7 +25,7 @@
       "hidden": false
     },
     "v2-prerelease": {
-      "release": "2.39.0",
+      "release": "2.40.0",

```

Comment:
@soninaren - do we need changes in this file as well?

Reply:
yes, it is a new feed that we need for certain versions for Visual Studio